src/lib.rs

```diff
@@ -797,23 +801,42 @@ pub enum BindingResource<'a> {
 }
 
 /// Operation to perform to the output attachment at the start of a renderpass.
-#[derive(Clone, Copy, Debug, Hash, PartialEq)]
+#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
+#[cfg_attr(feature = "trace", derive(Serialize))]
+#[cfg_attr(feature = "replay", derive(Deserialize))]
 pub enum LoadOp<V> {
     /// Clear with a specified value.
     Clear(V),
     /// Load from memory.
     Load,
 }
 
+impl<V: Default> Default for LoadOp<V> {
+    fn default() -> Self {
+        Self::Clear(Default::default())
+    }
+}
+
 /// Pair of load and store operations for an attachment aspect.
-#[derive(Clone, Debug, Hash, PartialEq)]
+#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
+#[cfg_attr(feature = "trace", derive(Serialize))]
+#[cfg_attr(feature = "replay", derive(Deserialize))]
 pub struct Operations<V> {
     /// How data should be read through this attachment.
     pub load: LoadOp<V>,
     /// Whether data will be written to through this attachment.
     pub store: bool,
 }
 
+impl<V: Default> Default for Operations<V> {
+    fn default() -> Self {
+        Self {
+            load: Default::default(),
+            store: true,
```

Comment:
I've seen [your comment](https://github.com/gfx-rs/wgpu-rs/issues/422#issuecomment-653777050) again and modified `store` to be `true` by default, as does upstream WebGPU.